src/smc-webapp/frame-editors/latex-editor/actions.ts

```diff
@@ -279,7 +295,7 @@ export class Actions extends BaseActions<LatexEditorState> {
       await patch_synctex(
         this.project_id,
         this.path,
-        force ? undefined : (time || this._last_save_time),
+        force ? undefined : time || this._last_save_time,
```

Comment:
Your change is technically right, because conditional has lower precedence than ||.  But I think it's very important to make this code *clear*, and conditional is literally [**just below** || in precedence... i.e., just barely](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence).   Also, the rest of the code in this file leaves in the parens to make the precedence clear.    It's like write `2 + (3 * 5)` instead of `2 + 3 * 5`.   Do you really find it more readable without the parens?

Reply:
well, it's the prettier magic. Personally, I'm also much more happy with rather more than less parens in logical expressions.

Reply:
Well I'm OK with just letting prettier do what it does in this particular case...